README.md

```diff
@@ -201,6 +201,42 @@ var railsImagePath = "<%= helpers.image_path('rails.png') %>";
 
 This is enabled by the `rails-erb-loader` loader rule in `config/webpack/shared.js`.
 
+## Ready for Elm
+
+To use Webpacker with [Elm](http://elm-lang.org), create a new app with `rails new myapp
+--webpack=elm` (or run `rails webpacker:install:elm` on a Rails app already setup with webpacker).
+The Elm library and core packages will be added via Yarn and Elm itself. An example `Main.elm` app
+is also added to your project in `app/javascript` so that you can experiment with Elm right away.
+For example, you might want to make the following changes to your newly generated, Elm-enabled,
+Rails app:
+
+### Routes (routes.rb)
+
+    Rails.application.routes.draw do
+      resource :demo, controller: :demo, only: :show
+    end
+
+### Controllers
+
+    class DemoController < ApplicationController
+      def show
+      end
+    end
+
+### Views
+
+Add an empty `app/views/demo/show.html.erb` file.
+Add `<%= javascript_pack_tag "hello_elm" %>` to `app/views/layouts/application.html.erb`.
+
+### Launch
+
+In two terminal windows, launch each of the following:
+
+    rails s
+    bin/webpack-dev-server
+
+Now you can open `http://localhost:3000/demo` in your default browser and see Elm running on Rails!
```

Comment:
That's true it is Rails-specific. Yeah, moving to another section of the README is a good idea since it applies to all. I'd like to experiment with more because if there is a way to do this automatically without manual steps, that'd be ideal.